scripts/audit_pdf.py

```
@@ -0,0 +1,554 @@
+                for i, page in enumerate(reader.pages):
+                    page_text = page.extract_text()
+                    jp_count = len(self.jp_core_pattern.findall(page_text))
+                    word_count = len(page_text.split())
+
+                    page_details.append({
+                        'page_number': i + 1,
+                        'word_count': word_count,
+                        'japanese_chars': jp_count,
+                        'text_length': len(page_text)
+                    })
+        except Exception as e:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)